separator_models/porous_separator.py

```diff
@@ -269,8 +280,8 @@ def output(self, axs, solution, an, ca, SV_offset, ax_offset):
 
         # Axis 5: Li+ concentration:
         Ck_elyte_an = solution[an.SVptr['C_k_elyte'][0]+SV_offset,:]
-        axs[ax_offset+1].plot(solution[0,:]/3600,
-            Ck_elyte_an[an.index_Li_elyte,:], label="an interface")
+        axs[ax_offset+1].plot(solution[0,:]/3600, Ck_elyte_an[an.index_Li,:],
+            label="an interface")
```

Comment:
I think you might not be working on the most recent update? This key should be `an.index_Li_elyte`, now, not `an.index_Li`.  Can you rebase onto `main` and force-push to this PR?